public/app/ui/pages/static/home.html

```diff
@@ -1,20 +1,37 @@
 <article id="home">
     <h1>Research Western Australian History</h1>
     <p>
-    Access to the Royal Western Australian Historical Society's collection for research and public interest purposes.
+        This website provides access to the Royal Western Australian Historical Society's collection for research and public interest purposes. Searches can be performed on the <a href="/photographs/search">photographic</a> and <a href="/museum/search">museum</a> collections, as well as <a href="/memorials/search">memorial records</a> and the <a href="/library/search">library catalogue</a>.
```

Comment:
I don't think the "This website..." lead-in is required or even desirable, but we will get proper text for here, from the client.